h2o-core/src/main/java/water/H2ONode.java

```diff
@@ -36,6 +36,7 @@
   transient short _unique_idx; // Dense integer index, skipping 0.  NOT cloud-wide unique.
   transient boolean _announcedLostContact;  // True if heartbeat published a no-contact msg
   transient public long _last_heard_from; // Time in msec since we last heard from this Node
+  transient public long _last_heard_from_rest; // Time in msec since somebody accessed the '3/Heartbeat' endpoint on this node
```

Comment:
IMHO this is misleading because it only applies to SELF, it shouldn't be defined on H2ONode in my opinion.

Reply:
Good point, will figure out better place